server/console_account.go

```diff
@@ -426,10 +437,14 @@ func (s *ConsoleServer) ListAccounts(ctx context.Context, in *console.ListAccoun
 				ID:       uuid.FromStringOrNil(user.Id),
 				Username: user.Username,
 			}
-			break
+			foundLimit = true
```

Comment:
Agreed - it's one extra row processing but cleaner code. 
I think there's only one side-effect with your proposal: in the case of rows > limit, the cursor would be created in the next `rows` iteration, pointing to the `limit+1` row instead of `limit`. Thus we'd need to change its paginated queries to be inclusive of the cursor id: `WHERE id >= $1` instead of `WHERE id > $1`. OK?

Reply:
Added a pointer to the previous element so we can keep queries intact (stuff like `AND (collection, user_id, read, key) > ($1, $2, $3, $4)"`)